astropy/convolution/tests/test_kernel_class.py

```diff
@@ -292,7 +292,7 @@ def test_custom_1D_kernel_list(self):
         Check if CustomKernel works with lists.
         """
         custom = CustomKernel([1, 1, 1, 1, 1])
-        assert custom.is_bool == True
```

Comment:
Agree with @pllim that the test should be for `is True`, so that one automatically checks that it is a `Bool`